Cargo.toml

```diff
@@ -10,6 +10,13 @@ license = "MIT / Apache-2.0"
 keywords = ["clipboard"]
 exclude = ["/.travis.yml"]
 
+[features]
+default = ["X", "wayland"]
+
+X = ["x11-clipboard"]
```

Comment:
I've named options after https://gentoo.org/support/use-flags/

Reply:
I don't think I've ever seen an uppercase Rust feature flag. So I'd use `x`, instead of `X`.

I also question the name `x` in general, since that isn't very descriptive. Our module is called `x11`, so the obvious feature name choice would be `x11` to me. So no `x11` feature -> no `x11_clipboard` module. Easy.